env/bin/Activate.ps1

```diff
@@ -0,0 +1,247 @@
+<#

```

Comment:
Looks like all the files from the env/ directory got into GitHub because the filter in .gitignore reads venv/.  
Two ways to fix this...
* an easy way but good only for small amounts of data is to update the .gititnore, then `git rm -rf venv/`.   Then recreate the virtual environment to match `env/`
* The better way, especially if you have large amounts of data that snuck in... find the sha using `git log` of the point before the data was added.  Create a branch from there, and add only the files you want.  Then discard the old branch.  That way the git history no longer tracks that extra files were inserted.